### DISCUSSION TITLE
Alternative HUD Styles?

### DISCUSSION THREAD

**Discussion by Sonic-TV** [emoose/OutRun2006Tweaks#242] [Suggestions]
### Describe the feature you'd like to see added

Would it be possible to use OutRun2 (squared) HUD style for the game?

**Comment by Tenome** [comment]
Got a screenshot example?

**Comment by Sonic-TV** [comment]
![image-28](https://github.com/user-attachments/assets/17ce6624-ed8d-4e9b-b070-ed432ffe31d1)
You don't need to change the kerning of bottom speedo numbers though
and if I remember correctly there are leftovers in the game files

**Comment by Sonic-TV** [comment]
> Weren't there discussions about being able to change the font?

I probably forgot that if there is....
